tutorials/0. Theory.ipynb

```diff
@@ -0,0 +1,270 @@
+{
```

Comment:
<p>We can solve most of these problems by simply borrowing modern tensor-processing frameworks like&nbsp;<code>TensorFlow</code>&nbsp;or&nbsp;<code>PyTorch</code>: they allow us to seamlessly integrate most of GPUs/TPUs into our workflow, greatly increasing the speed of inference. -&gt;</p><p>When we use apply neural networks for solving PDEs , we can solve most of these problems by simply borrowing modern tensor-processing frameworks like&nbsp;<code>TensorFlow</code>&nbsp;or&nbsp;<code>PyTorch</code>: they allow us to seamlessly integrate most of GPUs/TPUs into our workflow, greatly increasing the speed of inference.</p><p> </p><p>Anyways, it's a great passage!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/pydens/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>